idioms/default.md

```diff
@@ -22,11 +22,15 @@ different names, but there can only be one `Default` implementation per type.
 ```rust,ignore
 // note that we can simply auto-derive Default here.
 #[derive(Default)]
+use std::{path::PathBuf, time::Duration};
```

Comment:
remove the derive statement above (line 24) which is an error.
Than you can remove the ignore on line 22, since the test pass now! :D

Reply:
Also you should remove comment on line 23 since it's duplicated